layers/+lang/ocaml/packages.el

```diff
@@ -81,7 +81,13 @@
       (spacemacs/declare-prefix-for-mode 'tuareg-mode "mE" "errors")
       (spacemacs/declare-prefix-for-mode 'tuareg-mode "mg" "goto")
       (spacemacs/declare-prefix-for-mode 'tuareg-mode "mh" "help")
-      (spacemacs/declare-prefix-for-mode 'tuareg-mode "mr" "refactor"))))
+      (spacemacs/declare-prefix-for-mode 'tuareg-mode "mr" "refactor")))
+    (use-package merlin-iedit
```

Comment:
I am not sure that you need this `use-package` here, because `merlin` package [already comes][1] with `merlin-iedit` file. But maybe you'll need to add explicit `require`.

  [1]: https://github.com/melpa/melpa/blob/master/recipes/merlin

Reply:
I replaced it with an explicit `require` in `:config` section instead (so it loads it after `merlin` itself is loaded).